google/cloud/storage/testing/storage_integration_test.cc

```diff
@@ -44,36 +44,55 @@ StorageIntegrationTest::MakeIntegrationTestClient() {
   return MakeIntegrationTestClient(TestRetryPolicy());
 }
 
+google::cloud::StatusOr<google::cloud::storage::Client>
+StorageIntegrationTest::MakeBucketIntegrationTestClient() {
+  if (UsingTestbench()) return MakeIntegrationTestClient();
+
+  auto constexpr kInitialDelay = std::chrono::seconds(5);
+  auto constexpr kMaximumBackoffDelay = std::chrono::minutes(5);
+  auto constexpr kBackoffScalingFactor = 2.0;
+  return MakeIntegrationTestClient(
+      LimitedTimeRetryPolicy(/*maximum_duration=*/2 * kMaximumBackoffDelay)
```

Comment:
Is the 2 the kBackoffScalingFactor?

Reply:
I think what I wrote now is clearer, let me know.